### ISSUE TITLE
"Click to see video" iframe links not working

### ISSUE THREAD

**Issue by neonacorns** [saylordotorg/text_principles-of-marketing-v2.0#1]
The "Click to see video" links [[query](https://github.com/saylordotorg/text_principles-of-marketing-v2.0/search?utf8=%E2%9C%93&q=%22click+to+see+video%22)] try to open an inline iframe which is just blank. Right-clicking and opening the video in a new tab/window works fine.

Chrome 52, OSX 10.10.5
Ditto Safari, Firefox

Switching youtube links to /embed/ vs /v/ doesn't work. Possible solution: regex replacement to remove iframes and make it a regular link.

**Comment by jazinheira** [comment]
Yep, we need to replace that iFrame shenanigans. A regular link would work.

**Comment by jazinheira** [comment]
Ok! Replaced.
